instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/otlp/OtlpMetricExporterAutoConfiguration.java

```diff
@@ -28,26 +31,25 @@
 public class OtlpMetricExporterAutoConfiguration {
 
   @Bean
-  @ConditionalOnMissingBean
-  public OtlpGrpcMetricExporter otelOtlpGrpcMetricExporter(OtlpExporterProperties properties) {
-    OtlpGrpcMetricExporterBuilder builder = OtlpGrpcMetricExporter.builder();
-
-    String endpoint = properties.getMetrics().getEndpoint();
-    if (endpoint == null) {
-      endpoint = properties.getEndpoint();
-    }
-    if (endpoint != null) {
-      builder.setEndpoint(endpoint);
-    }
-
-    Duration timeout = properties.getMetrics().getTimeout();
-    if (timeout == null) {
-      timeout = properties.getTimeout();
-    }
-    if (timeout != null) {
-      builder.setTimeout(timeout);
-    }
-
-    return builder.build();
+  @ConditionalOnMissingBean({OtlpGrpcMetricExporter.class, OtlpHttpMetricExporter.class})
```

Comment:
What about if the user adds another type of span exporter (for logging by example)?